### ISSUE TITLE
Is it possible to trigger spiderfied using an anchor link tag

### ISSUE THREAD

**Issue by ghost** [jawj/OverlappingMarkerSpiderfier#129]
Basically, I have a list of the locations aside the map and I'm trying to trigger the spiderfied of the location nearby the one that I have click on the list. The list is using anchor tag elements.

**Comment by jawj** [comment]
Yes, if I've understood correctly that should be possible. You would want to trigger the `click` event on the relevant marker, conditional on that being a marker that will spiderfy when clicked (which you can establish with a call to `markersNearMarker(marker, true)`).
